packages/react-sdk/docusaurus/docs/React/02-tutorials/03-livestream.mdx

```diff
@@ -1,6 +1,339 @@
 ---
 title: Livestream Tutorial
-description: How to build a livestream experience using Stream's video SDKs
+description: How to build a live stream experience using Stream's React Video SDKs
 ---
 
-TODO
+## Quickstart
+In this quickstart, we'll build a live streaming experience that's similar to Twitch.
+
+- The live stream will run on Stream's Edge network of servers around the world
+- You can use ultra-low latency (WebRTC) based live streaming or HLS (slight delay, but better buffering)
+- We'll show how to publish to the live stream from your browser
+- Stream enables you to scale to millions of viewers
+- You can have multiple active participants in the live stream
+
+The app that we are going to build would work in two modes: Host (presenter) and Viewer (audience).
+
+The host will be able to publish the live stream and the viewers will be able to watch the live stream.
+If you want to see a fully-fledged example, check out this [sample app.](https://github.com/GetStream/stream-video-js/tree/main/sample-apps/react/livestream-app)
+
+Let's get started.
+
+## Step 1 - Preparations
+
+Before going into the code, let's make sure we have everything we need.
+
+- Add Stream's React Video SDK to your project:
+  - `yarn install @stream-io/video-react-sdk` or
+  - `npm install @stream-io/video-react-sdk`
+- Setup a Stream account at [GetStream's Dashboard](https://dashboard.getstream.io) or use your existing one
+  - We will need your API key and secret
+
+## Step 2 - Setup the host mode
+
+Hosts should be communicating with each other in real-time. For that purpose, we would use the standard Stream Video Call component, based on WebRTC.
+
+:::note
+In this tutorial, we will skip the details on how a `StreamVideoClient` can be set up. You can read more about it on the following page:
+[Authenticating your Users](../../guides/01-authentication/)
+:::
+
+In the snippet below, we create a connection to the Stream Edge server for the live stream host, we join the call and we publish our audio and video to the live stream.
+In this example, we use the predefined `<PaginatedGridLayout>` for the live stream.
+
+In case you want to have a more custom layout, you can build your own. See [Building Custom Layouts.](../ui-cookbook/participant-list/)
+
+
+```tsx
```

Comment:
I am in no way able to judge the code itself. I feel like it's very long though and might be hard to follow in one bit. Do we have any chances of breaking things up a bit to not make it seem so long?

If not and it becomes unclear or harder to follow, then just keep it as is!

Very naive idea:
We could break it up and extract the `BackstageUI` component into a separate code block, quickly explaining what it does and what it is for...

Reply:
@DaemonLoki I couldn't make the snippet shorter than this without losing important information. 

@MartinCupela I intentionally excluded a screenshot, as this tutorial doesn't build a UI, instead, it shows how a UI can be built.
At the end of the tutorial, the reader can see a live preview of our sample app, and play with it.

Reply:
I propose to add the links to the preview also at the beginning. I think it can help the understanding what is then read, because the reader can associate what was seen and experienced with the actual code. At least for me it always worked better.

Reply:
The link to the source code is also available on the top of the tutorial but it isn't made that highlighted as the links at the end.

Reply:
What if the Tip banner from the bottom was moved to the top?

Reply:
I'm not sure if we want to promote the App first. If we drag the readers to the app first, they might not know what they are looking at.
The guide shows an approach to how a lives stream app can be built. Then we provide a sample implementation.

@DaemonLoki, what is your take on this? What does more sense?